recipes/genesis4/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "genesis4" %}
+{% set version = "4.6.4" %}
+
+{% set build = 0 %}
+# ensure mpi is defined (needed for conda-smithy recipe-lint)
+{% set mpi = mpi or 'mpich' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/svenreiche/Genesis-1.3-Version4/archive/v{{ version }}.tar.gz
+  sha256: c53ba86343bebd965ce3c358e3db13a4ebb3f47d6e0590427fb82a7593f004dc
+
+build:
+  number: {{ build }}
+
+  # add build string so packages can depend on mpi variants explicitly:
+  # `pkg * mpi_mpich_*` for mpich
+  # `pkg * mpi_*` for any mpi
+
+  {% set mpi_prefix = "mpi_" + mpi %}
+  string: {{ mpi_prefix }}_h{{ PKG_HASH }}_{{ build }}
```

Comment:
```suggestion
  string: {{ mpi_prefix }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
```

Reply:
Resolved in [7dbf3f4](https://github.com/conda-forge/staged-recipes/pull/24644/commits/7dbf3f41f5545dcbc0f6e6cc567e7b6076fd6678)